### DISCUSSION TITLE
default prior

### DISCUSSION THREAD

**Discussion by NcyFS** [lindeloev/mcp#142] [General]
Hello,

First of all, thank you for this wonderful package. I have a question about the default prior.  What is it?

**Comment by lindeloev** [comment]
You can see the default prior for your model in `fit$prior`, e.g., `mcp::demo_fit$prior` It's JAGS code. it stays close to the priors used in the `brms` package. The next version (mcp 0.4) will bring an almost perfect correspondence for one-segment models but mcp 0.3 is pretty close.

Sources for more info:
 * [The JAGS manual](https://people.stat.sc.edu/hansont/stat740/jags_user_manual.pdf)
 * The default priors for the change point parameter(s) are explained in [the mcp article (preprint)](https://osf.io/preprints/fzqxv/)
 * [This article on mcp priors](https://lindeloev.github.io/mcp/articles/priors.html) on the mcp website.
 * The documentation of the "prior" argument in mcp() via `?mcp` in R or [here](https://lindeloev.github.io/mcp/reference/mcp.html).
 * Read up on brms default priors :-)

**Comment by NcyFS** [comment]
thank you so much for these information !
